pkg/setting/setting.go

```diff
@@ -1351,6 +1354,12 @@ func readSecuritySettings(iniFile *ini.File, cfg *Cfg) error {
 	return nil
 }
 
+func readAuthGrafanaComSettings(iniFile *ini.File, cfg *Cfg) (err error) {
```

Comment:
nit: this function currently never returns an error, so it could just have an empty return to simplify the code.